plugins/chat/app/jobs/regular/chat/update_chat_threads_enabled.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+module Jobs
+  module Chat
+    class UpdateChatThreadsEnabled < Jobs::Base
+      def execute(args = {})
+        has_thread_channels = ::Chat::Channel.where(threading_enabled: true).exists?
```

Comment:
This should be more afficient:

`Chat::Channel.exists?(threading_enabled: true)`